### DISCUSSION TITLE
Button for swapping height and width values under resolution options when "custom" is selected

### DISCUSSION THREAD

**Discussion by k1logr4m** [mcmonkeyprojects/SwarmUI#209] [Ideas]
I personally switch between portrait and landscape a lot. Every time I need to manually type the number because picking from the aspect ratio drop-down menu changes the resolution. I'd like to have a button that can quickly perform the task without typing.

**Comment by mcmonkey4eva** [comment]
requests go on the issues page https://github.com/mcmonkeyprojects/SwarmUI/issues

**Reply by k1logr4m** [reply]
I apologize. I'm not familiar with GitHub yet, usually I just download assets. I've opened up an issue.
